tests/test_coreg.py

```diff
@@ -109,6 +110,51 @@ def test_bias(self):
         # Check that the original model's bias has not changed (that the _meta dicts are two different objects)
         assert biascorr._meta["bias"] == bias
 
+    def test_all_nans(self):
+        """Check that the coregistration approaches fail gracefully when given only nans."""
+        dem1 = np.ones((50, 50), dtype=float)
+        dem2 = dem1.copy() + np.nan
+        affine = rio.transform.from_origin(0, 0, 1, 1)
+
+        biascorr = coreg.BiasCorr()
+        icp = coreg.ICP()
+        
+        pytest.raises(ValueError, biascorr.fit, dem1, dem2, transform=affine)
```

Comment:
Does this function just check that the method biascorr.fit with arguments dem1, dem2 and tranform raises a ValueError??

Reply:
Yes!